<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>changeUserPWD</class>
 <widget class="QMainWindow" name="changeUserPWD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QFrame" name="frame_2">
      <property name="styleSheet">
       <string notr="true">#frame_2{border:2px solid rgb(170,85,0)}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="2">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QFrame" name="frame">
           <property name="styleSheet">
            <string notr="true">#frame{border:2px solid rgb(170,85,0)}</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout">
                  <item>
                   <widget class="QLabel" name="label">
                    <property name="minimumSize">
                     <size>
                      <width>111</width>
                      <height>51</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>111</width>
                      <height>51</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <family>宋体</family>
                      <pointsize>14</pointsize>
                     </font>
                    </property>
                    <property name="text">
                     <string>原密码：</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_2">
                    <property name="minimumSize">
                     <size>
                      <width>111</width>
                      <height>51</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>111</width>
                      <height>51</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <family>宋体</family>
                      <pointsize>14</pointsize>
                     </font>
                    </property>
                    <property name="text">
                     <string>新密码：</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_3">
                    <property name="minimumSize">
                     <size>
                      <width>111</width>
                      <height>51</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>111</width>
                      <height>51</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <family>宋体</family>
                      <pointsize>14</pointsize>
                     </font>
                    </property>
                    <property name="text">
                     <string>确认密码：</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_2">
                  <item>
                   <widget class="QLineEdit" name="lineEdit_yuanmima">
                    <property name="minimumSize">
                     <size>
                      <width>351</width>
                      <height>51</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>351</width>
                      <height>51</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <family>宋体</family>
                      <pointsize>12</pointsize>
                     </font>
                    </property>
                    <property name="echoMode">
                     <enum>QLineEdit::Password</enum>
                    </property>
                    <property name="placeholderText">
                     <string>请输入原密码</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="lineEdit_new">
                    <property name="minimumSize">
                     <size>
                      <width>351</width>
                      <height>51</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>351</width>
                      <height>51</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <family>宋体</family>
                      <pointsize>12</pointsize>
                     </font>
                    </property>
                    <property name="echoMode">
                     <enum>QLineEdit::Password</enum>
                    </property>
                    <property name="placeholderText">
                     <string>请输入新密码</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="lineEdit_confirnew">
                    <property name="minimumSize">
                     <size>
                      <width>351</width>
                      <height>51</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>351</width>
                      <height>51</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <family>宋体</family>
                      <pointsize>12</pointsize>
                     </font>
                    </property>
                    <property name="echoMode">
                     <enum>QLineEdit::Password</enum>
                    </property>
                    <property name="placeholderText">
                     <string>必须与新密码一致</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="minimumSize">
                   <size>
                    <width>111</width>
                    <height>51</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>111</width>
                    <height>51</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>宋体</family>
                    <pointsize>14</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>验证码：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_yanzhengma">
                  <property name="minimumSize">
                   <size>
                    <width>175</width>
                    <height>51</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>175</width>
                    <height>51</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>宋体</family>
                    <pointsize>14</pointsize>
                   </font>
                  </property>
                  <property name="placeholderText">
                   <string>请输入验证码</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QPushButton" name="pushButton">
           <property name="minimumSize">
            <size>
             <width>141</width>
             <height>51</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>141</width>
             <height>51</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>宋体</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>确认修改</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>847</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
